src/NumericalAlgorithms/SphericalHarmonics/Python/StrahlkorperFunctions.cpp

```diff
@@ -0,0 +1,27 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "NumericalAlgorithms/SphericalHarmonics/Python/StrahlkorperFunctions.hpp"
+
+#include <pybind11/pybind11.h>
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "NumericalAlgorithms/SphericalHarmonics/Strahlkorper.hpp"
+#include "NumericalAlgorithms/SphericalHarmonics/StrahlkorperFunctions.hpp"
+
+namespace py = pybind11;
+
+namespace ylm::py_bindings {
+void bind_strahlkorper_functions(pybind11::module& m) {  // NOLINT
+  m.def(
+      "cartesian_coords",
+      [](const ylm::Strahlkorper<Frame::Inertial>& strahlkorper) {
```

Comment:
[optional] I've actually found myself wanting this exact function in c++ several times (where you only pass a strahlkorper and get back the coords). Add it in c++ as well then bind that?

Reply:
ok done